modules/virt-adding-vm-to-service-mesh.adoc

```diff
@@ -61,7 +61,7 @@ To add a virtual machine (VM) workload to a service mesh, enable automatic sidec
 +
 [NOTE]
 ====
-To avoid port conflicts with sidecars, do not use ports used by Istio proxy for user workloads.
+To avoid port conflicts, do not use ports used by the Istio sidecar proxy. These include ports 15000, 15001, 15006, 15008, 15020, 15021, and 15090.
```

Comment:
I wonder if this is the best location for this information in the assembly. Should keeping these ports available be a prerequisite?